mathesar/tests/rpc/columns/test_c_base.py

```diff
@@ -123,11 +76,14 @@ def mock_column_info(_table_oid, conn):
             'valid_target_types': None
         }
     ]
+    mocked_exec_msar_func.fetchone.return_value = [expect_col_list]
     actual_col_list = columns.list_(table_oid=23457, database_id=database_id, request=request)
+    call_args = mocked_exec_msar_func.call_args_list[0][0]
     assert actual_col_list == expect_col_list
+    assert call_args[2] == table_oid
```

Comment:
I think you should check all the arguments. That way, if someone fatfingers a function name down the road (i.e., messes up the string describing the SQL function), or accidentally switches the order of the connection and function name, this will catch it. Same for the other similar tests.

Also, we should use something more identifiable than `True` for the mock connect yield. Maybe something like:

```python
class MockConnection:
    pass
```

Reply:
> I think you should check all the arguments. That way, if someone fatfingers a function name down the road (i.e., messes up the string describing the SQL function), or accidentally switches the order of the connection and function name, this will catch it.

@mathemancer These conditions are already covered by `test_db_wrapper`. Not only will it check whether the 2nd argument in each `exec_msar_func` call is a string, it'll also verify that a function specified in the string exists on the underlying database. This would mean that we would also catch any accidentally switches of order between the connection and function name.

Reply:
Ahhhh you're right. I forgot about that.